### ISSUE TITLE
[Bug] F1 bug

### ISSUE THREAD

**Issue by RickyTheRacc** [MeteorDevelopment/meteor-client#1915]
### Describe the bug

Entering or exiting any gui will seemingly at random not unhide the HUD if "Hide Hud" is enabled in the gui tab. Also, when entering a world the HUD will be hidden by default, where it should instead be visible

https://medal.tv/games/minecraft/clips/7nDWoGbZ5WX3I/d1337tFw0HW3?invite=cr-MSxsaXQsMzQwMjE5MDEs

### Steps to reproduce

```bash
Enable "hide hud" in the Gui tab and open and close your inventory or the meteor clickgui. Observe how it sometimes fails to reenable the HUD. As well as this, log in and out of a server or singleplayer world and observe how the hud is hidden by default when you join.
```

### Meteor Version

1396

### Minecraft Version

1.18.1

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by RickyTheRacc** [comment]
EDIT: This bug only happens when entering any other tab than the Clickgui. If you only use the clickgui and then close out, F1 behaves as expected. Switching to the config, baritone, HUD tabs all cause the HUD not to unhide.

**Comment by RickyTheRacc** [comment]
@MineGame159 Please fix 😭

**Comment by arlomcwalter** [comment]
> Fixed in b53f5de3e17c6ff4abf21f83b5785f044255b7ca.

that wont work properly

**Comment by MineGame159** [comment]
> > Fixed in [b53f5de](https://github.com/MeteorDevelopment/meteor-client/commit/b53f5de3e17c6ff4abf21f83b5785f044255b7ca).
> 
> that wont work properly

7fde0624e8c008914960c3b7ff411e8c44527b6c
